### DISCUSSION TITLE
ScoreV1 in Lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26960] [Ideas]
This is the first time im doing a post/discussion on github so i apologize if there is anything wrong and **i may be wrong on this!**

I want to start by mentioning something important that i think the osu! devs might have to take into consideration when working on Lazer since Lazer will eventually replace osu! stable, Ranked score farmers on osu! stable have played osu! for gaining ranked score for years to achieve certain goals like 100 billion ranked score, 500 billion, 1 trillion etc and its all been done with ScoreV1 since if you use ScoreV2 on stable its unranked. Since all set scores on stable use ScoreV1 the total ranked score number shown on your osu! profile is a very specific number and if lazer will have a completely new scoring system im guessing it would have to recalculate that number into a completely new one making all the progress people have made for years kind of all for nothing since many players only play for certain goals involving ranked score like i mentioned.

So i think a solution for this is for players to have a option to use ScoreV1 in lazer. The ''Classic'' score display isnt exacly the same as ScoreV1 because i tested it and the scores i saw on the same map where different.

**Comment by bdach** [comment]
I don't think user total ranked score will be retroactively recalculated.

I do not foresee us adding score V1 to lazer because that causes many other issues.

**Reply by ghost** [reply]
Alright thanks for the answer.

**Reply by WitherFlower** [reply]
> I don't think user total ranked score will be retroactively recalculated.

Can you go into more detail as to what you mean by this ?
Ranked Score totals are currently wrong in lazer, as they haven't been updated following score conversion, and score farmers are hoping to see the updated values once mod multiplier balancing has been settled. I hope you don't mean that this recalculation will not happen.

**Reply by bdach** [reply]
> Can you go into more detail as to what you mean by this ?

Well my thinking was that score farmers are presumably going to be majorly pissed off by us changing their score total using converted lazer score which is after all an approximation. If that is not the case then maybe we can do that.

**Reply by WitherFlower** [reply]
After asking scorefarmers, it seems that most of them either don't care or are waiting for the change to be made.
Should I make a separate issue/discussion to gather all the issues that need to be sorted out for the recalculation ?

**Reply by bdach** [reply]
> gather all the issues that need to be sorted out for the recalculation

i guess, if there are any more that i'm not aware of

**Reply by moosepi** [reply]
I understand that V1 itself wouldn't be feasible, but would it be possible to introduce other map-specific variables (e.g. max combo count, difficulty multiplier) to the classic scoring calculation to make the NM SS cap closer to stable? For reference I've found that `objcount * 300 + maxcombo * objcount * diffmult * 6` works pretty well for standard, although this is just from eyeballing it -- I haven't run any kind of large scale testing.

As it stands while the general scaling works quite well I find that a lot of harder/longer/high slider count maps tend to lose a significant amount of score from v1 -> classic. Some (admittedly cherry-picked, but I found each within just a few f2s so IMO it's not that uncommon) examples with the scores for the global NM top play, which are all high acc FCs, and value for SS with the proposed formula:

| Map | V1 | Current classic | Proposed |
|--|--|--|--|
| https://osu.ppy.sh/beatmapsets/1455997#osu/2992795 | 47119400 | 30988981 | 45610800 |
| https://osu.ppy.sh/beatmapsets/192896#osu/458582 | 184936620 | 134600429 | 187813620 |
| https://osu.ppy.sh/beatmapsets/437683#osu/942356 | 48700386 | 34701424 | 49350756 |
| https://osu.ppy.sh/beatmapsets/1779346#osu/3644079 | 171758280 | 118864859 | 177122700 |
| https://osu.ppy.sh/beatmapsets/770240#osu/1619425 | 57758500 | 35045171 | 56105820 |

and for more extreme examples:

| Map | V1 | Current classic | Proposed |
|--|--|--|--|
| https://osu.ppy.sh/beatmapsets/724192#osu/1529189 | 1571805970 | 1256917582 | 1594253580 |
| https://osu.ppy.sh/beatmapsets/933630#osu/1949106 | 1811653120 | 1498773325 | 1815358770 |
| https://osu.ppy.sh/beatmapsets/162800#osu/1200520 | 1201954170 | 977904050 | 1192679928 |

(sorry if this would be better as a separate post but seems relevant here?)

**Reply by ghost** [reply]
> After asking scorefarmers, it seems that most of them either don't care or are waiting for the change to be made. Should I make a separate issue/discussion to gather all the issues that need to be sorted out for the recalculation ?

I really care and i think alot of other people will care too. I recently learned about how lazer will de-value all stable scores by something by 96 and i really dont think that is good.

**Reply by WitherFlower** [reply]
@Flaaavia 

> I really care and i think alot of other people will care too. 

I directly asked the core of the scorefarming community and have not felt pushback, people are eager to see the total being updated, as right now, Ranked Score is literally broken (it's supposed to be the sum of all of one's top lb scores). Maybe what can be done for players like you is splitting lazer and stable's Ranked Score totals. 

> I recently learned about how lazer will de-value all stable scores by something by 96 and i really dont think that is good.

The discussion about mod multipliers isn't really meaningful, as a 0.96× nerf to all scores does not change the ordering on the Ranked Score leaderboard.

@moosepi 

Classic scoring was adjusted to be proportional to object_count^2, it might be possible to improve it with the difficulty multiplier, but that would also add a fair bit of complexity to the code.

**Reply by ghost** [reply]
> I directly asked the core of the scorefarming community and have not felt pushback, people are eager to see the total being updated, as right now, Ranked Score is literally broken (it's supposed to be the sum of all of one's top lb scores).

Alright.

> Maybe what can be done for players like you is splitting lazer and stable's Ranked Score totals.

I would gladly like to see that happen.

> The discussion about mod multipliers isn't really meaningful, as a 0.96× nerf to all scores does not change the ordering on the Ranked Score leaderboard.

Oh ok.

**Reply by GreenCatWalkOnWaterHotDogAirFlygInLava** [reply]
@WitherFlower 

Personally I dont want to get involved but if we dont then.. "have not felt pushback" happens. Lets start from the beginning, why are we using the current score system in Lazer?

The reason I ask is because I have a feeling that we should redo everything from the absolute beginning. However I only read some random information on the osu! website and Youtube and might be missing something important here.

I have big score many many hundred billions hopefully im "core" enough even if im casual and would insult scorefarmers by calling myself that. (the core part seemed important so I had to mention it).

Edit: I would interpret score farmers still using osu! classic instead of Lazer that give you many benefits like more score.. and free access to osu! direct and all the other benefits as a big sign that they are actually pretty annoyed. But im only guessing based on that alone.

**Reply by WitherFlower** [reply]
@GreenCatWalkOnWaterHotDogAirFlygInLava

I'm not sure I understand what you're trying to say here. Lazer's current scoring was made from the ground up. There are also many parity issues between stable and lazer so I don't know if you can conclude anything from your observation of score farmers still using stable.

I also think it's fair to mention that there are score farmers like me that have in fact switched to lazer.

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
True there are many resons why people dont migrate to Lazer. But for a
score farmer it should be very tempting to do so asap no matter other
issues. If you cherrypick Github users it wouldnt be surprising if the
majority use Lazer already I would in fact expect that to be the case.

What im trying to say is that its very possible that the thought process
when creating the new system from the ground didnt ask the question "why"
enough times and therefore gave us a poor tool to work with. A hammer is
good but you dont use it to cut bread, why would you.

Its like when I tried Lazer last time and I  had to click many more times
to do the exact same thing as you did in classic. It came to my
understanding then that people are not always making rational changes.

Let me ask you this: Why are we even using score to measure performance in
e-sport tournaments instead of PerformancePoints that was designed for
exactly just that?

One of the few arguments I heard for using the new score system was that
its supposed to be easier for tournament casters to manage. But it come
with a cost and side effects. It dont even make no sense, score is not a
way to measure skill. And we dont need three values (pp,score,%) to show
pretty much the same thing.

Den fre 15 mars 2024 00:36WitherFlower ***@***.***> skrev:

**Reply by bdach** [reply]
I'm sorry but I don't think there's a productive discussion to be had here unless you bring up *specific* grievances. I can't do anything with anything that you've said in here.

It's all well and good to wave hands and say "let's do away with score and pp and accuracy altogether and do a grand unified design" but what *else* do you propose even? If we went out and said that in earnest as developers with no real concrete plan we would be *crucified* by pretty much every side in that respect. In fact most of the last year or so of lazer development was *undoing* simplifications that we did to all of the systems involved because players didn't like them.

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
Im very glad to hear you say that because what if we dont make any change
for the sake of change and instead keep the old dynamic, fun score system
instead of forcing a boring, static one that dont take length or difficulty
into consideration at all?

Im just doing my part here expecting nothing because "Have not felt
pushback". Now im giving you exactly just that pushback. Let me ask you
this instead, what would it take for you guys to change your mind?

If you dont know this already then its very unlikely that you can change
your mind. Lazer is a big ship that is hard to turn around for sure, too
much hard work put into it.. prestige and so on.

Part of the reason why you dont get so much pushback might be because the
Lazer project have gone on for so long. Wait another 20 years and you might
get no pushback at all.




Den fre 15 mars 2024 11:50Bartłomiej Dach ***@***.***> skrev:

**Reply by bdach** [reply]
I still don't know what you're talking about here. It seems like you have a specific issue in mind but instead of saying _directly_ what the problem is you're dancing around it and throwing vagueries instead.

Specific - complaints - please. If you're not willing to provide them, I'm not willing to engage with this unproductive "pushback" any longer.

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
Oh but I have, ill try keep it simple and direct then read carefully: the
problem is the 1million cap that dont take difficulty or length (combo)
into consideration.

End of discussion I guess.. no point in communicating in bad faith. But you
can proudly say next time the topic resurface that at least one guy gave
pushback and that he wrote that the score system in Lazer can go multiply
with a goat.

Den lör 16 mars 2024 08:36Bartłomiej Dach ***@***.***> skrev:

**Reply by bdach** [reply]
Score is not intended to be a measure of beatmap difficulty to begin with. It's intended to convey how well you did on this map.

I'm not sure if you're aware but on score farmers' request, "classic" scoring - an approximation of score V1 - is used for user profile totals. It's also available in the client if you so desire.

Debating the score system is months to years late now. Several discussions took place about this. Many community members were involved. Maybe brilliant from-the-ground-up designs should have been proposed *before* we went live with leaderboards, but of course "nobody cared about lazer back then" did they? There's just no winning on our side.

**Reply by ghost** [reply]
If what you got from the player reaction to the new scoring and mechanics over the last months and years was that you're good to go, you're honestly in an echo chamber. Players have been complaining from the start. There were always easy ways to gauge the real community sentiment, like in game banners. Of course you don't "win" if you just don't bother.

**Reply by bdach** [reply]
- tens of community meetings  https://osu.ppy.sh/wiki/en/Community/osu!_community_meetings
- involving multiple community figures like https://youtu.be/OVccI1arosM
- various twitter polls that actually were listened to (https://twitter.com/ppy/status/1670984707469873153, https://twitter.com/ppy/status/1749415464743444611)
- various discord polls

if after all that we're still in an "echo chamber" then i don't know what to even say anymore

**Reply by ghost** [reply]
Could you show me the polling about whether slider accuracy should even be implemented as default or at all? Because from what I've seen, Peppy has completely dismissed all complaints and concerns and has handwaved it all away with the Classic mod, which now has its own set of issues. What about the score cap of 1m? Is that what the average player wants, for the entire game to use tournament scoring? I'd love to see those polls, maybe I'm wrong and the whole community loves it, though I would be very surprised. 

The more you try to shoehorn in these changes, the more complicated it gets to try to not break everything. Is it really worth it?

**Reply by peppy** [reply]
> Could you show me the polling about whether slider accuracy should even be implemented as default or at all? 

Was discussed in community meetings linked above, amongst other locations.

And as always, stable remains as it is so you can choose to not embrace change by staying on stable.

**Reply by ghost** [reply]
So you made the decision based off of the input of only a handful of people, there was no community polling, and you were either unaware of player sentiment or just didnt care. I would've been surprised if there had been polling, having closely followed development for years. Thanks for the confirmation. "We aren't going to ruin Stable, so don't worry about it" isn't a valid response when it comes to the future of the game, but I wouldn't expect any more from osu devs. Good luck. o7

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
I agree, but then you are only looking at the individual map not a map in
relation to others so S****ks 30s map is worth as much as Doomsdays ZUN
Medely and I cant get it into my head how any scorefarmer could be ok with
that. Heretics.

I also agree that its very late to make changes now. But for further
similar situations I recommend that the information spread and the
reasoning behind the changes are conveyed in a better manner. Otherwise the
(lets change word) feedback from the community will be less, of lower
quality and less constructive. (I still dont understand the reasoning if
there ever where any).

I remember in the beginning when a few where interested in the new score
system. People used to try it out in multiplayer lobbies but that dont
happen nowdays. I havent seen that happen for years now so how wanted is
this change really.

There have ofcourse allways been people who care about Lazer. Lazer is an
amazing thing but it cold be considered both a cure and a poison.

Yes im aware of the approx thing but not its origin. So there been a
discussio in the SF Discord? Sounds like some kind of comporomise where
made.

You know.. some people sit still in the boat when its shaking, hope for bad
things to never become a reality and for good sense to prevail. Only to
react when the boat is already upside down and they float in the cold water
cursing whoever made it happen.


Den lör 16 mars 2024 11:45Bartłomiej Dach ***@***.***> skrev:

**Reply by WitherFlower** [reply]
> I agree, but then you are only looking at the individual map not a map in
relation to others so S****ks 30s map is worth as much as Doomsdays ZUN
Medely and I cant get it into my head how any scorefarmer could be ok with
that. Heretics.

Lazer already solves that with classic scoring. See it for yourself.

### 30 second map
![osu_2024-03-16_15-25-41](https://github.com/ppy/osu/assets/52672543/1b7976eb-ccca-448a-a8a8-f6e5b965c560)

### 30 minute map
![osu_2024-03-05_17-02-07](https://github.com/ppy/osu/assets/52672543/85ed91ff-d851-4d0d-b70d-2d2c4e65bc8f)

### Settings menu
![image](https://github.com/ppy/osu/assets/52672543/634a0406-3c89-4569-9b6a-670925ddede4)

**Reply by ghost** [reply]
A toggle to change how the score looks doesn't change the core issue, that the devs have ruined a fun system (classic scoring) and fun gameplay mechanics (slider leniency) in exchange for something that only a subset of players were even asking for. Then when discussion and complaints go nowhere, they come back with, "We gave you what you wanted, a toggle to try to ignore that the game is ruined. What are you complaining about now?" The community wouldn't even ask for all of these "toggles" anywhere near this often if they felt listened to in the first place, people were requesting toggles for the new osu logo ffs.

**Reply by WitherFlower** [reply]
Please explain why you think that the old scoring has been ruined. We can't answer you if you don't explain the exact issue you have.

**Reply by GreenCatWalkOnWaterHotDogAirFlygInLava** [reply]
In wait for the answer from lainosu I believe that part of that have been explained already but ill throw a solution and compromise to the problem on the table.

Make the old score system the default one and keep the new. Anyone who want to use the new can sit there and actively click on the "mod" icon every time. If there after a few years are the most used system then make it the default one.

How would you feel about this?


Regarding the sliderhead change (im ofcourse against that too) it indeed make the game more accurate but you can just increase OD to get that effect. 
The side effect of this will be that new players get even more flooded with misses than before. The game have an unlimited difficulty already but you increase the lowest difficulty for the sake of being accurate.
Its not without reason we see more use of sliders in easier difficulties.

Same situation here, are there any arguments to warrant a change?

**Reply by WitherFlower** [reply]
I don't understand where this discussion is going. 
I will ask again, for one last time : Please explain exactly what the problem with the scoring sysyem is. I will stop responding to this discussion until I have a clear answer because this is just becoming noise at this point.

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
Then you need to be clear what information you want and in what format
because you recieved the information several times.

If you want another answer it might be good to change the question or
rephrase it, asking the exact same thing over and over and expecting
another answer makes me want to quote Albert Einstein "insanity, over and
over" but it could also be that you actually dont want an answer and that
would make sense. You are happy with how things are and dont wany it to
change.. right?

The problem is the score system in Lazer.

That alone doesent say much and its not the root problem because the person
who created the system and why it was created and later implimented is..

If you want a short summary of the problem:

1. Very few wanted/want to use it.
2. Its boring.
3. It raise the lowest difficulty.
4. Doesent consider diff/length.
5. It was forced onto players in a tyrannical way.
6. It make most players have to select a mod meaning you increasing number
of clicks for no good reason.


Den mån 18 mars 2024 15:07WitherFlower ***@***.***> skrev:

**Reply by GreenCatWalkOnWaterHotDogAirFlygInLava** [reply]
@Respektive Sorry for randomly pulling you in here. I have a question about the compromise of the score system that where made. Are you genuinely ok with it or did you go along with it because it was "the lesser evil"? Did you feel preassure to agree to it?

Leading question here, I dont even know if you where involved, I only assume.

**Reply by respektive** [reply]
Dunno why my opinion in particular matters here but :shrug: 

I was helping initially with finding some values to make the classic scoring match scorev1 pretty closely, and I ran some queries on the data we have on osu! alt to get some idea of the ranked score sums of players.
I was always a fan of a scoring system that can be calculated from just the score info and does not require a replay to go with it, so yeah I am genuinely ok with it. I think the 1mil cap works well for individual map leaderboards comparison and for score farming the latest classic scoring formula matches scorev1 so closely, I don't see any problem with it. The only problem there is right now are the mod multipliers, but that is not a problem with the scoring system itself and the multipliers can still be adjusted and applied to old scores, so I'm not that worried about that.

If i were having any issues with it, I don't think I would be playing exclusively on lazer ever since the ranked update.

**Comment by ZeroAurora** [comment]
Just wandering around and reading the scoring code before seeing this discussion. And I was just chatting with friend about "the dark side of combo-heavy scoring". What a coincidence.
Scoring has always been a heated topic in lazer's development. It started with a lazer-specific algorithm and changed to a ScoreV2 variant last year. Some like and some dislike. No one can conclude that "it's boring" or "many hate it". The community needs a survey and lazer **is** the survey.
No one has the right to represent a whole, large community. Data are there to prove whether the scoring system is the choice, and community meetings are there to push the system forward. If you are still hating it, lazer is still beta quality and not a production, nor replacing stable. And stable will be around for many years. No one pushes you to adopt an unripe system and you have the choice. But the new thing needs players to test out and give feedback, before the old thing fading away.
And one thing to remind: this discussion may be too heated that it is going out of its original scope.

**Comment by mooshiros** [comment]
As a score farmer, I don't think we necessarily need scorev1 specifically (i.e. keeping sliderhead accuracy wouldn't really matter) to be added, just a scoring system that scales with your combo in the  same way it does in stable. As it is now, lazer classic scoring makes FCs give less score than they should and non-FCs give more score than they should. 

For example, my 4.1k combo (with other combos over 1k throughout the map run on The Unforgiving is 400 million score on stable while 700 million on lazer classic scoring. Kingling's hdhr FC on The Unforigiving is 1.6 billion score on stable while only 1 billion on lazer classic scoring.

But as it is currently, lazer completely breaks score farming (and this is disregarding the glitch where you can play a map on stable after you already played it on lazer and get full score from your stable play, essentially allowing you to double the amount of score you get from one map).

**Reply by WitherFlower** [reply]
If I'm understanding correctly, your main issue is that non-FCs get too much score compared to scoreV1, is that correct ?
If that's the case, let me ask you this : do you think it's fair to lose half of the score by only missing once in the middle of the map ? This is how scorev1 behaved, and many people didn't think that was fair, so it's one of the things lazer scoring addresses. Maybe the nerf on miss penalty was too strong, but I definitely think that scorev1 isn't the right solution.

**Reply by mooshiros** [reply]
@WitherFlower No, it's not fair, but whether it's fair or not is irrelevant to my (and most score farmers') issue. I don't care whether or not lazer scoring is used for map leaderboards, in fact I think it should be used for map leaderboards. The issue is with the overall score leaderboards. The entire point of score farming is combo game, it's fun to grind for FCs on maps (especially long ones) and get high combo an get rewarded for it. The only thing I ask is that at the end of each play when you see that little green text telling you how much score got added to your ranked score, it's scorev1 and not lazer scoring. Most people don't care about their total ranked score, and the people that do (like me) want it to be scorev1. My issue is also that some maps get nerfed in the amount of score they give (i.e. unforgiving hdhr fc losing an insane 600 million score), which also makes score farming less fun.

Also, let's be real, the only reason 99% of people dislike that missing in the middle of the map kills your score is because of scorev1ing, which is already being fixed (as it basically has to be fixed for CSR to work).

**Comment by GreenCatWalkOnWaterHotDogAirFlygInLava** [comment]
@ZeroAurora Think I wrote it earlier but, we have had the option of
changing to ScoreV2 for a long time now in osu!, people have tried it out.
I played quite alot of multiplayer in open lobbies and I havent seen anyone
use ScoreV2 in many years. People where curious first when it came out but
that interest was short lived. If players change the score system they
change it to "% Accuracy" not "ScoreV2" that never happens and whoever
belive otherwise is somewhat disconnected from osu!. It is true that some
players dont like the combo system, some prefer the accuracy but ScoreV2 is
completely irrelevant unless its forced or set as default (sly way to force
a change). Why even make a poll about this, *just log the use of ScoreV2 in
the osu! client and you will have the correct and completely unbiased data*.

You get diffrent answers depending on how you ask a quesiton and who. Now
the opinion of new players are very important but in this case id have to
say that it might be a problem to ask this category of players about their
opinion because.. the majority of players have an opinion but a big
percentage might not know what score is or how they get it, what they care
about is PP this is the main metric for a new player because it give them
an instant gratification. ScoreV2 is likely easier for a new player to
understand. But that takes away the mystique, they have other things to
focus on anyway they can figure out how it works later. Its like when youre
on a date or whatever, you dont tell the person everything about yourself
its strategically a smart move to be a bit mysterious because its more
interesting. Tell everything and you kill the mystique so from that point
of view the score system is good the way it is.


@respektive I was just curious, but everyones opinion matter we all know
that and.. you know your thing. In fact I have no problem saying that your
opinion is more valued than mine I dont see that as a problematic thing to
say, its just honest.


@mooshiros The balance thing does not really effect the gameplay or the
fun. Im not interested in how the system is balanced so if you get 400
million score instead of 700 million  isnt the question for me as long as
it is scaling enough, as you mention. But I can see how people would want
it to be in one way or another.

Even if the sliderhead change seem minor, its a minor change in the wrong
direction. osu! already have an unlimited difficulty and most who try
playing the game are likely to give up. With that in mind, we dont have to
spam players with even more misses for the sake of being "correct" thats
why im opposed to that as well. That and the argument we all know, that its
handy to use sliders on wierd timings to make the map easier to play. It
kinda blow my mind how someone thought this would be a good idea to
increase the lowest difficulty and make the game a little bit harder to get
into.

Tried to answer. If I missed something just poke me or DM in game, my
vision is getting worse its hard to jump from word to word and from one
line to another it took over an hour to write this I have to check every
word several times and im sure I missread something anyway.

Den ons 7 aug. 2024 kl 01:38 skrev mooshiros ***@***.***>:

**Reply by mooshiros** [reply]
@GreenCatWalkOnWaterHotDogAirFlygInLava The reason my post mentioned only the balance of score is that my ONLY gripe with lazer scoring is that it ruins score farming. That's all I want fixed. As someone who has been playing this game for over 7 years, I also prefer sliderhead leniency, however I recognize that from the standpoint of designing a rhythm game (and also from the standpoint of new players) it makes far more sense to have your judgement on a slider be determined by your tapping accuracy (which is how LNs work in every other rhythm game) rather than how many slider ticks you missed or whether you hit the sliderend or not. Players may be used to the latter, but it makes objectively more sense. Osu is not on a date with its players, confusing incoming players to keep the "mystique" is just an excuse for bad game design imo. Lazer scoring *is* better than scorev1, however it ruins an entire subset of the community that *depends* on scorev1 for their playstyle (score farming), and that's why we should have a scorev1-like system to submit to be added to your ranked score. I don't care if the per-map leaderboards use it or not, but I do care that the overall ranked score leaderboards (and ideally total score/levels) use it rather than the current lazer classic scoring.

**Comment by mooshiros** [comment]
I want to just add on to this that the same issue is present in catch, and while score farming isn't as prevalent there it still is a thing people do (shoutouts to syu)

**Comment by mooshiros** [comment]
Not sure if this is the right place to put this, but on a related note I think something should be put in place to avoid destroying the effort put in by the leaderboard farming community. Leaderboard farming isn't broken by lazer scoring, but it *is* broken by the classic mod multiplier and by rate changes above 1.5 being on the leaderboard. Even legendary scores like Griffin's My Love Perma-#1 or Tactic's Perma-#1 on Disco Prince are now easily beatable on lazer, which I think is very unfair to the leaderboard farming community. It could even be done on a map-by-map basis, with all maps ranked before a certain date not taking into account the 0.95 classic mod multiplier and not allowing scores above 1.5x on the leaderboard, but maps ranked after that date acting like lazer leaderboards do right now. This would mean that all of the effort put in by the leaderboard farming community thus far won't be erased and all they have to do is switch to lazer for the newly ranked maps, while still keeping the new lazer system (though in all honesty 0.95 multipiler for classic is still an insane nerf)

**Reply by felipemarins** [reply]
Something similar (#25751) has been proposed before and was rejected. I also don't think that wiping every score above 1.5x rate change for maps on a certain date range is the best solution. As for the multiplier, the 0.96x is just a placeholder and it will change when the devs have the tools to change it ready.
